dockerfile/anaconda-live-iso-creator/lmc-build

```diff
@@ -1,3 +1,10 @@
+# ======================================
+# WARNING!
+# THIS FILE IS GENERATED FROM A TEMPLATE
+# DO NOT EDIT THIS FILE MANUALLY!
+# ======================================
+# The template is located in: lmc-build.j2
+
 #!/bin/bash
```

Comment:
This will not work, I think. The hashbang needs to be special-cased in the rendering script :-/

Reply:
See https://github.com/rhinstaller/anaconda/pull/5104

Reply:
Shebang handling ported to f39 in #5108 - feel free to ack+merge and rebase this on top (or not if it's wrong!).